tools/certificate.sh

```diff
@@ -48,8 +48,15 @@ if [ -z "${SERVICE}" ]; then
     exit 1
 fi
 
-[ -z "${NAMESPACE}" ] && NAMESPACE=fluid-system
-[ -z "${CERT_DIR}" ] && CERT_DIR=/opt/certs
+if [ -z "${NAMESPACE}" ]; then
+    echo "'--namespace' must be specified"
+    exit 1
+fi
+
+if [ -z "${CERT_DIR}" ]; then
+    echo "'--certDir' must be specified"
+    exit 1
+fi
```

Comment:
why must be specified without permitting default?

Reply:
As we discussed offline, it's easier and makes it more explicit